QUARKUS-1079.md

```diff
@@ -0,0 +1,74 @@
+# QUARKUS-1079 - Hibernate Reactive as Tech Preview
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1079
+
+For native mode similar test coverage as for JVM mode needs to be ensured.
+
+## Scope of the testing
+This feature added previews of two extensions: 
+ - Hibernate Reactive
+ - Hibernate Reactive with Panache
+
+Following test coverage should be implemented for Hibernate Reactive:
+- Usage of application.properties file for passing reactive-hibernate settings [examples]
```

Comment:
Why is `[examples]` text in your Test Plan items?

Reply:
It describes, which test suite contains these tests. It's either [examples], which means "examples" section in framework, or [testsuite], which means "new" test suite